src/Agent/NewRelic/Agent/Core/JsonConverters/LogEventWireModelCollectionJsonConverter.cs

```diff
@@ -100,6 +101,20 @@ private static void WriteJsonImpl(JsonWriter jsonWriter, LogEventWireModelCollec
                     jsonWriter.WriteValue(logEvent.TraceId);
                 }
 
+                if (logEvent.ContextData?.Count > 0)
+                {
+                    jsonWriter.WritePropertyName(Attributes);
```

Comment:
I assume it's ok that we're reusing the property name "attributes" here? (It's also in the metadata above.)

Reply:
I believe so. In fact, they should match. The outer "attributes" are global attributes, the inner ones are specific to log events.